src/markdown-pages/docs/managers/core-manager.md

```diff
@@ -3,12 +3,144 @@ path: "/docs/managers/core-manager"
 
 
+## <a name="dependencies"></a>Including the Manager in a test
+
+To use the Core Manager in a test you must import the _@CoreManager_ annotation into the test, as shown in the following example: 
+
+```
+@CoreManager
+public ICoreManager coreManager;
+```
+
+You also need to add the Manager dependency into the pom.xml file if you are using Maven, or into the build.gradle file if you are using Gradle. 
+
+If you are using Maven, add the following dependencies into the _pom.xml_ in the _dependencies_ section:
+
+```
+<dependency>
+<groupId>dev.galasa</groupId>
+<artifactId>dev.galasa.core.manager</artifactId>
+</dependency>
+```
+
+If you are using Gradle, add the following dependencies into ```build.gradle``` in the _dependencies_ closure:
+
+```
+dependencies {
+compileOnly 'dev.galasa:dev.galasa.core.manager'
+}
+```
+
+
+Finally, use the annotation that is provided by the Manager in the test. The annotation calls the Manager at runtime. In this example, the provided annotation is _@CoreManager_. 
+
+
+# <a name="configuring"></a>Configuring 
+
+The following are properties used to configure the Core Manager:
+
+## <a name="cps"></a>Configuration Properties
+
+<details>
+<summary>Resource String Pattern CPS Property</summary>
+
+| Property: | Resource String Pattern CPS Property |
+| --------------------------------------- | :------------------------------------- |
+| Name: | core.resource.string.[length].pattern |
+| Description: | Sets the patterns of resources strings for a length of string.  The patterns are from the Galasa ResourcePoolingService which uses a homegrown syntax.  |
+| Required:  | No |
+| Default value: | {A-Z} for each byte for the specified length |
+| Valid values: | For each character can be a constant or a random choice from a literal, eg {A-Z results in a single character between A and Z inclusive. {0-9} or {a-zA-Z0-9} are options. DFH{A-Z}{0-1}{0-9}{0-9}{0-9}, results in DFHA1789 for example, the 4th character can only be 0 or 1. 
+| Examples: | <code>core.resource.string.8.length={A-Z}{A-Z}{A-Z}{A-Z}{A-Z}{A-Z}{A-Z}{A-Z}<br> </code> |
+
+</details>
+
+# <a name="annotations"></a>Annotations provided by the Manager
+
+The following annotations are provided by the Core Manager:
+
+<details>
+<summary>Logger</summary>
+
+| Name: | Logger |
+| --------------------------------------- | :------------------------------------- |
+| Name: | @Logger |
+| Description: | Creates the log which is then automatically stored in the Result Archive Store (RAS) by the Galasa framework. |
```

Comment:
we should document what type of object can be annotated with this annotation

Reply:
updated to say that an object of type `log` can be annotated with this annotation.